src/modules/CdmSourceList/components/List/ModalCreateEdit/presenter.jsx

```diff
@@ -61,7 +62,7 @@ function ModalCreateEdit(props) {
     label: 'Cancel',
   };
 
-  const fields = getDataSourceCreationFields(dbmsTypeList);
+  const fields = getDataSourceCreationFields(dbmsTypeList, false, isStandalone && props.initialValues.centralId);
```

Comment:
The last function parameter is named `isStandalone` and you are passing `isStandalone && props.initialValues.centralId` into it, which is not just flag of standalone/network mode. Let's name it in a more self-descriptive way which matches the actual meaning, e.g.:

```
// src/const/dataSource.js
getDataSourceCreationFields(dbmsTypeList, useOnlyVirtual = false, disabledFields = {})

//src/modules/CdmSourceList/components/List/ModalCreateEdit/presenter.jsx
const fields = getDataSourceCreationFields(dbmsTypeList, false, { name: isStandalone && props.initialValues.centralId });
```